generator settings/rolePresentation.md

```diff
@@ -0,0 +1,28 @@
+Our generator uses tables for controlling the layout, mostly because some of the email clients not support padding ang margin styling on <div> tags.
+You can find more information about this here: https://blog.edmdesigner.com/html-email-padding-margin-border/
+
+For accessibility reasons, we built in the generator a new setting, called rolePresentation. It adds a presentation role to the table that used for layout purposes. 
```

Comment:
I would highlight the word **accessibility**.

Reply:
oh, I just checked the cheatsheet, how to formatting the text, I'll using it from now!